src/app/custom-editor/custom-editor.component.html

```diff
@@ -30,20 +31,28 @@
         <ng-template #unfocused>
           <svg xmlns="http://www.w3.org/2000/svg" width="10" height="10" fill="currentColor"
                viewBox="0 0 16 16">
-            <path d="m10.79 12.912-1.614-1.615a3.5 3.5 0 0 1-4.474-4.474l-2.06-2.06C.938 6.278 0 8 0 8s3 5.5 8 5.5a7 7 0 0 0 2.79-.588M5.21 3.088A7 7 0 0 1 8 2.5c5 0 8 5.5 8 5.5s-.939 1.721-2.641 3.238l-2.062-2.062a3.5 3.5 0 0 0-4.474-4.474z"/>
-            <path d="M5.525 7.646a2.5 2.5 0 0 0 2.829 2.829zm4.95.708-2.829-2.83a2.5 2.5 0 0 1 2.829 2.829zm3.171 6-12-12 .708-.708 12 12z"/>
+            <path
+              d="m10.79 12.912-1.614-1.615a3.5 3.5 0 0 1-4.474-4.474l-2.06-2.06C.938 6.278 0 8 0 8s3 5.5 8 5.5a7 7 0 0 0 2.79-.588M5.21 3.088A7 7 0 0 1 8 2.5c5 0 8 5.5 8 5.5s-.939 1.721-2.641 3.238l-2.062-2.062a3.5 3.5 0 0 0-4.474-4.474z"/>
+            <path
+              d="M5.525 7.646a2.5 2.5 0 0 0 2.829 2.829zm4.95.708-2.829-2.83a2.5 2.5 0 0 1 2.829 2.829zm3.171 6-12-12 .708-.708 12 12z"/>
           </svg>
         </ng-template>
-        <span class="status-text">{{editorFocused ? 'Focused' : 'Unfocused'}}</span>
+        <span class="status-text">{{ editorFocused ? 'Focused' : 'Unfocused' }}</span>
       </div>
     </div>
     <div class="right-side">
-      <select class="drop-down"
-              [(ngModel)]="currentView"
-              (ngModelChange)="onViewChange($event)">
-        <option *ngFor="let view of editorViewsConst; let index = index"
-                [selected]="editorViewsConst[index] === currentView"
-                [ngValue]="editorViewsConst[index]">{{view|titlecase}}</option>
+      <select
+        class="drop-down"
+        *ngIf="availableViews"
+        [(ngModel)]="currentView"
+        data-cy-editor="viewDropDown"
+        (ngModelChange)="onViewChange($event)"
+        [title]="'Current view: ' + currentView"
```

Comment:
This is not consistent with what you used in `ngx-monaco-editor`. There you did it this way. Please choose one and keep it consistent. 
```suggestion
        title="Current view: {{currentView}}"
```

Reply:
Fixed

Reply:
Is it? I still see `[title]="'Current view: ' + currentView"` in the changes

Reply:
I've changed it all to the [title], instead of also using the title={{}}

Reply:
In the monaco one you could also do ```[style.height]="height - 43 + 'px'"```